snapcraft/plugins/_ros/rospack.py

```diff
@@ -18,12 +18,15 @@
 import logging
 import subprocess
 import tempfile
-from typing import List, Set
+from typing import List, Set, TYPE_CHECKING
 
-import snapcraft
 from snapcraft.internal import common, repo
 from snapcraft import formatting_utils
 
+if TYPE_CHECKING:
+    from snapcraft.project import Project
```

Comment:
why conditional?

Reply:
the type is only used for type checking and quoted where used.

Reply:
I guess I mean, why change this at all? :)  I've done a similar annotation and am curious what the problem was?

Reply:
Oh, I cannot remember now and this probably was not needed in what the code ended up looking like :man_shrugging: